styles.css

```
@@ -0,0 +1,117 @@
+body {
+  background-color: burlywood;
+}
+.content {
+  display: flex;
+  flex-direction: column;
+  width: 680px;
+  padding: 24px 24px;
+  background-color: white;
+  font-family: sans-serif;
+  margin-left: auto;
```

Instead of giving `margin-left` and `margin-right` separately, you can use the shorthand property.

```suggestion
  margin: 0 auto;
```

---

Done